client/components/language-picker/README.md

```diff
@@ -0,0 +1,69 @@
+LanguagePicker
+==============
+
+React component used to display a Language Picker.
+
+---
+
+## Example Usage
+
+```js
+import LanguagePicker from 'components/language-picker';
+
+const languages = [
+	{
+		value: 1,
+		langSlug: 'en',
+		name: 'English',
+		wpLocale: 'en_US',
+		popular: 1
+	},
+	{
+		value: 11,
+		langSlug: 'cs',
+		name: 'Čeština',
+		wpLocale: 'cs_CZ'
+	}
+];
+
+class LanguagePickerDemo {
+	onSelectLanguage = ( event ) => {
+		this.setState( { language: event.target.value } );
+	}
+
+	render() {
+		return (
+			<LanguagePicker
+				languages={ languages }
+				valueKey="langSlug"
+				value={ this.state.language }
+				onChange={ this.onSelectLanguage }
+			/>
+		);
+	}
+}
+```
+
+---
+
+## LanguagePicker
+
+The `LanguagePicker` is a form component with API similar to a HTML `input` or `select`
+element - its value is specified in a `value` prop, and it calls an `onChange` handler
+when its value changes.
+
+#### Props
+
+`languages` - **required** Array with information about languages, their names, language
+slugs, popularity etc. It's exported by `config` module under the `languages` key.
+
+`valueKey` - **optional** Which property of the `languages` objects should be used
+as the `value` of the component? The default is `value`, other options are `langSlug` and
+`wpLocale`. This determines what kind of value is expeced as the `value` property and
+what is the value passed as parameter in the `onChange` handler.
```

Comment:
excpeced -> expected.

We normally describe what the prop is or does ( e.g. "Determines which property of the `languages` objects should be used…"

You could say it more succinctly like "Should be one of `value` (default), `langSlug` or `wpLocale`". (as a bonus, it also suggests the [propType syntax](https://github.com/reactjs/prop-types/blob/b06161d16f553bfbf006fc3d23f22531771efe3e/factoryWithTypeCheckers.js#L66))

More generally, this is pretty confusing. What's the backstory here? Is "kind of value" the type? There's a comment in the code about expecting numbers sometimes, is this about that?